ronanboiteau/django_safe_template_engine
# Base safe template engine project

- Add a template rendering engine `SafeTemplateEngine` to render untrusted template code
    - Support a few safe filters and tags
- Add pytest tests to:
    - Make sure safe filters and tags work
    - Make sure unsafe tags and loader tags are blocked
- Add base CircleCI config to run tests
